projects/packages/my-jetpack/composer.json

```diff
@@ -23,12 +23,24 @@
 		],
 		"test-coverage": [
 			"@composer install",
-			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\"",
+			"pnpm install",
+			"pnpm run test -- --coverageDirectory=\"$COVERAGE_DIR\" --coverage"
```

Comment:
As written both of these will write to `$COVERAGE_DIR/clover.xml`, so the JS coverage will overwrite the PHP coverage. You should arrange for them to have different names.

Also, you may want to pass `--coverageReporters=clover` for the JS so it won't also generate an lcov report.

Reply:
@anomiex Yes! I was seeing now the crash between PHP and JS, pushing a fix for that.

Reply:
done.